.github/workflows/integration_tests.yml

```diff
@@ -433,7 +436,9 @@ jobs:
         timeout-minutes: 45
         continue-on-error: true
         env:
+          VERSION: ${{ env.VERSION }}
```

Comment:
This should not be needed. `env.VERSION` indicates that this is already available as a global environment variable, which contains the `matrix.VERSION` (which contains the version of the Kubernetes platform provider)

Reply:
good catch - has been removed